projects/igniteui-angular/src/lib/grids/pivot-grid/pivot-data-selector.component.ts

```diff
@@ -170,7 +170,7 @@ export class IgxPivotDataSelectorComponent {
         this.dims = [
```

Comment:
Since all collections could potentially be set to null same should be applied to the other collections or you can use the `allDimensions` property in the pivot grid.

Reply:
The requested changes are done